src/guide/list.md

```diff
@@ -1,8 +1,8 @@
-# List Rendering
+# Renderizado de Lista
 
-## Mapping an Array to Elements with `v-for`
+## Mapeando una matriz a Elementos con `v-for`
```

Comment:
What do you think about this?

```suggestion
## Mapeando un Arreglo a Elementos con `v-for`
```

Reply:
Thanks, @migsalazar! I forgot we're using `arreglo` instead of `matrix`.